Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md

```diff
@@ -0,0 +1,3 @@
+# Example1 - Background color changer
+
+<img src="example1.gif"/>
```

Comment:
Use Code blocks.

Reply:
Required changes made in the latest commit

Reply:
Still looks the same?